static/js/publisher/release/actions/releases.js

```diff
@@ -128,7 +128,10 @@ export function releaseRevisions() {
     ) {
       progressive = pendingRelease.progressive;
 
-      if (progressive.key === null) {
+      if (
+        progressive.key === null ||
+        progressive.key.indexOf("ui-temp") === 0
```

Comment:
It may be worth putting "ui-temp" into a constant - it's already in 3 places I think.